File: HelloWorld/HelloWorld-Fortran/HelloWorld.f90
program hello

  include 'mpif.h'

  integer mpierr, rank, procs

  call MPI_Init ( mpierr )
  call MPI_Comm_size ( MPI_COMM_WORLD , procs , mpierr )
  call MPI_Comm_rank ( MPI_COMM_WORLD , rank , mpierr )

  write (*,*) 'Hello world from ', rank, 'of', procs 

  call MPI_Finalize ( mpierr )

end program hello

File: 2015/HelloWorld/HelloWorld-C/HelloWorld.c
#include <stdio.h>
#include <mpi.h>

int main(int argc, char **argv) {
    int mpierr, procs, rank;
    mpierr = MPI_Init(&argc, &argv);
    mpierr = MPI_Comm_size(MPI_COMM_WORLD, &procs);
    mpierr = MPI_Comm_rank(MPI_COMM_WORLD, &rank);
    printf("Hello from %d of %d\n", rank, procs);
    mpierr = MPI_Finalize();
    return 0;
}
